providers/neo4j.js

```diff
@@ -0,0 +1,215 @@
+const { Provider, util } = require('klasa');
+const { v1: neo4j } = require('neo4j-driver');
+
+module.exports = class extends Provider {
+
+	constructor(...args) {
+		super(...args);
+		this.db = null;
+	}
+
+	init() {
+		const driver = neo4j.driver('bolt://localhost', neo4j.auth.basic('neo4j', 'test'));
+		this.db = driver.session();
+	}
+
+	/**
+	 * Checks if a table exists.
+	 * @param {string} table The name of the table you want to check.
+	 * @returns {Promise<boolean>}
+	 */
+	hasTable(table) {
+		return this.db.run(`MATCH (n:${table}) RETURN n`).then(data => Boolean(data.records.length));
+	}
+
+	/**
+	 * Create a collection within a DB. Options may be specfied, refer to Neo4j docs.
+	 * @param {string} table Name of the Collection to crate
+	 * @returns {Promise<*>} Returns a promise containing the created Collection.
+	 */
+	createTable(table) {
+		return this.db.run(`CREATE (n:${table}) RETURN n`).then(data => data.records.map(node => node._fields[0].identity.low))[0];
```

Comment:
You have `[0]` in the wrong place. Also, just do this

```javascript
return this.db.run(`CREATE (n:${table}) RETURN n`).then(data => data.records[0]_fields[0].identity.low);
```

Reply:
It cant be accessed like an array.